Makefile

```diff
@@ -9,9 +9,9 @@ S3_TARGET?=s3://public-repo-1.hortonworks.com/HDP/cloudbreak/
 #S3_TARGET=s3://public-repo.sequenceiq.com
 
 ifeq ($(GIT_BRANCH), release)
-FLAGS="-X main.Version $(VERSION)"
+FLAGS="-X main.Version=$(VERSION)"
```

Comment:
In the golang community every project uses space, so i wouldn't change that one.